sdk/events_scm.go

```diff
@@ -47,6 +47,8 @@ type GitLabProject struct {
 	Namespace         string `json:"namespace"`
 	Name              string `json:"name"`
 	PathWithNamespace string `json:"path_with_namespace"` //would be repo full name
+	WebURL            string `json:"web_url"`
+	VisibilityLevel   int    `json:"visibility_level"`
```

Comment:
How is this different for GitLab and GitHub?

Reply:
This doesn't exist in GitHub, it's private true/false.